src/reporting.rs

```diff
@@ -6,6 +6,7 @@ use std::backtrace::Backtrace;
 use std::borrow::Cow;
 use std::fmt::Write;
 use std::sync::Arc;
+use std::sync::atomic::{AtomicBool, Ordering};
```

Comment:
Please follow the project style

```suggestion
use std::sync::atomic::AtomicBool;
use std::sync::atomic::Ordering;
```

Reply:
Done, FWIW there's an unstable rustfmt option to enforce this if you're OK requiring unstable rustfmt features for contributions: https://rust-lang.github.io/rustfmt/?version=v1.8.0&search=granula#Item%5C%3A